pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -102,27 +121,9 @@ pub mod pallet {
 	pub type Seasons<T: Config> = StorageMap<_, Identity, SeasonId, SeasonOf<T>>;
 
 	#[pallet::storage]
-	#[pallet::getter(fn mint_available)]
-	pub type MintAvailable<T: Config> = StorageValue<_, bool, ValueQuery>;
-
-	#[pallet::type_value]
-	pub fn DefaultMintFee<T: Config>() -> BalanceOf<T> {
-		(1_000_000_000_000_u64 * 55 / 100).unique_saturated_into()
-	}
-
-	#[pallet::storage]
-	#[pallet::getter(fn mint_fee)]
-	pub type MintFee<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery, DefaultMintFee<T>>;
-
-	#[pallet::type_value]
-	pub fn DefaultMintCooldown<T: Config>() -> T::BlockNumber {
-		5_u8.into()
-	}
-
-	#[pallet::storage]
-	#[pallet::getter(fn mint_cooldown)]
-	pub type MintCooldown<T: Config> =
-		StorageValue<_, T::BlockNumber, ValueQuery, DefaultMintCooldown<T>>;
+	#[pallet::getter(fn configs)]
+	pub type Configs<T: Config> =
```

Comment:
Let's rename for consistency if we can:
```suggestion
	#[pallet::getter(fn season_configs)]
	pub type SeasonConfigs<T: Config> =
```